Automattic/jetpack
# Payment blocks: Always make available the "One-Time" subscription recurrence

Since the user is currently being able to create One-Time subscriptions from Calypso, we should allow the same behaviour from Payment blocks product management system (membership subscriptions).

This was already available, however gated by a conditional check. Since we should always allow users to create them, the condition was removed entirely.





Fixes #23632

#### Changes proposed in this Pull Request:

* Remove allowCreateOneTimeInterval property since we allow users to create One-Time subscriptions form Calypso /earn.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* WPCOM: Apply D77750-code on your sandbox
* Atomic: Switch to this branch
* Go to wp-admin/post-new.php
* Add a Premium-content/Payment button block
* Add a new subscription from the Inspector
* Make sure that you can create a "One-Time" Subscription from the "Interval" dropdown.
